lib/flag_shih_tzu.rb

```diff
@@ -343,6 +350,10 @@ def named_scope_method
         # Can't use respond_to because both AR 2 and 3 respond to both +scope+ and +named_scope+.
         ActiveRecord::VERSION::MAJOR == 2 ? :named_scope : :scope
       end
+
+      def is_active_record_class?
```

Comment:
Inconsistent indentation detected.<br>Rename `is_active_record_class?` to `active_record_class?`.